components/inputs/test/input-text.test.js

```diff
@@ -244,6 +244,33 @@ describe('d2l-input-text', () => {
 			expect(elem.value).to.equal('hello');
 		});
 
+		it('should change "value" property when input value changes because of blur event on edge', async() => {
+			const browserType = window.navigator.userAgent;
+			if (!(browserType.indexOf('Trident') > -1 || browserType.indexOf('Edge') > -1)) return;
+
+			const elem = await fixture(normalFixture);
+			getInput(elem).value = 'hello';
+			setTimeout(() => dispatchEvent(elem, 'blur', true));
+			await oneEvent(elem, 'change');
```

Comment:
I'm going to modify this test to make sure it fails when running in sauce, otherwise it's useless.